cmd/consensus.cpp

```diff
@@ -53,10 +52,12 @@ static const fs::path kBlockchainDir{"BlockchainTests"};
 
 static const fs::path kTransactionDir{"TransactionTests"};
 
-static const std::vector<fs::path> kExcludedTests{
-    // Very slow tests
+static const std::vector<fs::path> kSlowTests{
     kBlockchainDir / "GeneralStateTests" / "stTimeConsuming",
+    kBlockchainDir / "GeneralStateTests" / "VMTests" / "vmPerformance" / "loopMul.json",
+};
 
+static const std::vector<fs::path> kFailingTests{
```

Comment:
I believe @gumb0 is working on fixing these tests upstream.

Reply:
Yes but only nonce tests.
I believe the next test release will include nonce larger than 2^64-1 being forbidden.
But also the new rule: nonce == 2^64-1 is fobidden, too, see updated [EIP-2681](https://eips.ethereum.org/EIPS/eip-2681).